coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -15,18 +15,46 @@ $(document).ready(function documentReady() {
   });
 
   var sidebarToggle = false;
-  $('body').on('click', '#hideCourseSidenav', function () {
+  $("body").on("click", "#hideCourseSidenav", function() {
     sidebarToggle = !sidebarToggle;
-    var sidebarWidth = sidebarToggle ? $('.course-accordion').width() : 0;
+    var sidebarWidth = sidebarToggle ? $(".course-accordion").width() : 0;
 
-    $('.course-accordion').toggleClass('hide-sidenav');
-    $('.course-card').animate({'margin-left': sidebarWidth / 2 + 'px'}, 300);
-    $('.course-accordion').animate({'margin-left': '-' + sidebarWidth + 'px'}, 300);
+    $(".course-accordion").toggleClass("hide-sidenav");
+    $(".course-card").animate({ "margin-left": sidebarWidth / 2 + "px" }, 300);
+    $(".course-accordion").animate(
+      { "margin-left": "-" + sidebarWidth + "px" },
+      300
+    );
   });
 
-  $('.course-accordion .collapsible').clone().appendTo('#courseMobileSidebar');
-  $('.collapsible').collapsible();
-  $('body').on('click', '#closeSideNav', function () {
-    $('.button-collapse').sideNav('hide');
+  $(".course-accordion .collapsible").clone().appendTo("#courseMobileSidebar");
+  $(".collapsible").collapsible();
+  $("body").on("click", "#closeSideNav", function() {
+    $(".button-collapse").sideNav("hide");
+  });
+
+  var parentElementClass = ['item',
```

Comment:
@andela-johia  There are other classes that you have left out e.g those of the GCB components, how robust can this be?

Reply:
@andela-johia Consider the code snippet below, other than `<input type="text">` one should be able to click anywhere outside and navigate using the arrow keys. As you may see, there are many classes here that are not captured and many more as we go through the modules. What happens when a new module is added as we did for carousel in the oeditor? 
```
<div class="qt-sa-question qt-embedded" id="3xDGkRmSVXJP.0.6218562888794112">
    <div class="qt-points">
      <em>    
          1 point
      </em>
    </div>

    <div class="qt-question">Q1</div>

    <div class="qt-response">
        <input type="text">
    </div>

      <div class="qt-hint">
        <button class="gcb-button qt-hint-button">
           Hint 
        </button>
      </div>
    
    <div class="qt-feedback qt-hidden" role="alert">
    </div>
  </div>
```

Reply:
@chegejames I understand where you are coming from. So I should reverse engineer it instead so instead of looking for classes outside the content, I should target elements inside the content